src/common/uilocale.cpp

```diff
@@ -240,10 +467,31 @@ bool wxUILocale::UseDefault()
 }
 
 /* static */
-bool wxUILocale::UseLanguage(const wxLanguageInfo& info)
+bool wxUILocale::UseLocaleName(const wxString& localeName)
 {
-    wxUILocaleImpl* const impl = wxUILocaleImpl::CreateForLanguage(info);
-    if ( !impl )
+    wxUILocaleImpl* impl = NULL;
+    if (localeName.IsSameAs("C", false))
```

Comment:
Should we test for `"POSIX"` too here?

Reply:
Most likely yes, if _POSIX_ is associated with the standard C locale.